odoo/addons/base/views/ir_rule_views.xml

```diff
@@ -80,6 +80,9 @@
                     <filter string="Delete Access Right" name="delete_access_right" domain="[('perm_unlink', '=', True)]"/>
                     <separator/>
                     <filter string="Archived" name="inactive" domain="[('active', '=', False)]"/>
+                    <group expand="0" string="Group By">
```

Comment:
expand="0" ?

Reply:
Changed to expand="1"

Was not sure on its effect, got to know from

https://github.com/odoo/odoo/blob/3f9cb47cb6fdc240a58f3daa3bd4cf32f55cb084/addons/web/models/models.py#L66